baselines/fedrs/README.md

```diff
@@ -0,0 +1,122 @@
+
+**Training Hyperparameters:** Table below shows the **default** training hyperparams for the experiments. Values are from the original paper where provided (e.g. learning rate, momentum, weight decay)
+
+| Description | Default Value |
+| ----------- | ----- |
+| strategy | fedavg |
+| scaling factor for missing classes (alpha) | 0.9 |
+| fraction fit | 0.1 |
+| local epochs | 2 |
+| learning rate | 0.03 |
+| momentum | 0.9 |
+| weight decay | 5e-4 |
+| number of rounds | 1000 |
+| batch size | 64 |
+| num shards per partition | 5 |
+| model | vgg11 |
+
+## Environment Setup
+
+
+```bash
+# Create the virtual environment
+pyenv virtualenv 3.10.14 <name-of-your-baseline-env>
+
+# Activate it
+pyenv activate <name-of-your-baseline-env>
+
+# Install the baseline
+pip install -e .
+```
+
+If running with **wandb**:
+
+```
+wandb login
+```
+
+## Running the Experiments
+
+## Table 5 (Performance comparisons with standard FL algorithms (VGG11))
+
+### Commands for C10-100-5 (CIFAR-10 with 100 clients and 5 shards per client)
+
+|  | Command
+| ----------- | ----- | 
+| FedAvg (default) | `flwr run .` | 
+| FedProx (0.0001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.0001"` | 
+| FedProx (0.001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.001"` | 
+| FedRS ($\alpha$=0.5)| `flwr run . --run-config "alpha=0.5"` | 
+| FedRS ($\alpha$=0.9) | `flwr run . --run-config "alpha=0.9"` | 
+
+### Commands for C10-100-2 (CIFAR-10 with 100 clients and 2 shards per client)
+
+|  | Command
+| ----------- | ----- | 
+| FedAvg (default) | `flwr run . --run-config num-shards-per-partition=2` | 
+| FedProx (0.0001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.0001 num-shards-per-partition=2"` | 
+| FedProx (0.001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.001 num-shards-per-partition=2"` | 
+| FedRS ($\alpha$=0.5)| `flwr run . --run-config "alpha=0.5 num-shards-per-partition=2"` | 
+| FedRS ($\alpha$=0.9) | `flwr run . --run-config "alpha=0.9 num-shards-per-partition=2"` | 
+
+### Commands for C100-100-20 (CIFAR-100 with 100 clients and 20 shards per client)
+
+|  | Command
+| ----------- | ----- | 
+| FedAvg (default) | `flwr run . --run-config num-shards-per-partition=20 dataset='cifar100'` | 
+| FedProx (0.0001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.0001 num-shards-per-partition=20 dataset='cifar100'` | 
+| FedProx (0.001)| `flwr run . --run-config "alg='fedprox' proximal-mu=0.001 num-shards-per-partition=20 dataset='cifar100'` | 
+| FedRS ($\alpha$=0.5)| `flwr run . --run-config "alpha=0.5 num-shards-per-partition=20 dataset='cifar100'` | 
+| FedRS ($\alpha$=0.9) | `flwr run . --run-config "alpha=0.9 num-shards-per-partition=20 dataset='cifar100'` | 
+
+
+After 1000 server steps, the accuracy results are as follows:
+
```

Comment:
@flydump @jafermarq Given the commands for C10-100-2 and C100-100-20 are provided, maybe worth to complete the table with the more results?